packages/tss-client/src/client.ts

```diff
@@ -59,31 +51,12 @@ if (globalThis.js_read_msg === undefined) {
   };
 }
 
-globalThis.process_ga1 = async (tssImportUrl: string, msg_data: string): Promise<string> => {
-  const worker = new TssWebWorker(tssImportUrl);
-  const res = worker.work<string>("process_ga1", [msg_data]);
-  return res;
-};
-
 if (globalThis.js_send_msg === undefined) {
   globalThis.js_send_msg = async function (session: string, self_index: number, party: number, msg_type: string, msg_data?: string) {
     const tss_client = globalThis.tss_clients.get(session) as Client;
     tss_client.log(`sending msg, ${msg_type}`);
     if (msg_type.indexOf("ga1_data_unprocessed") > -1) {
-      globalThis
-        .process_ga1(tss_client.tssImportUrl, msg_data)
-        .then((processed_data: string) => {
-          tss_client.msgQueue.push({
-            session,
-            sender: party,
-            recipient: self_index,
-            msg_type: `${session}~ga1_data_processed`,
-            msg_data: processed_data,
-          });
-          return true;
-        })
-        .catch((err) => console.error(err));
-      return true;
+      throw new Error("ga1_data_unprocessed should not be sent directly");
```

Comment:
Don't throw an error here.

Reply:
Instead this whole if can be removed.

Reply:
The reason why this can be removed is if `"unsupported"` is received for message ` "ga1_worker_support"`, then this message is never emitted later on.

Reply:
I'm aware of this. I decided to throw an error, because, technically, this should never happen. And if it does, it shows that something is going wrong. I would be more willing to remove this if we don't have the worker message flow at all anymore.

Reply:
Ok, fine with me.